docs/contribute/colima.qmd

```diff
@@ -0,0 +1,130 @@
+# Colima setup for MacOS M1/M2 users
+
+If you are working with an `arm64` architecture (Mac M1/M2), you will need to
```

Comment:
```suggestion
If you are working with an M1 or M2 CPU architecture, you will need to
```

The parentheses don't really make sense here IMO unless we're talking about generic arm64 CPU architectures and I don't think we are given the title heading.